azavea/temperate
# Implement static text for Concerns

## Overview

Some weather events don't have applicable indicators, but we still want to show concerns for them. Examples include "storm surge" or "permanent inundation".

Static text/units are used if `Concern.indicator == None`.

### Demo

<img width="529" alt="screen shot 2018-03-12 at 10 31 36 am" src="https://user-images.githubusercontent.com/1818302/37289696-b0c288aa-25e0-11e8-8717-44813e303314.png">
<img width="404" alt="screen shot 2018-03-12 at 10 31 47 am" src="https://user-images.githubusercontent.com/1818302/37289699-b232ecca-25e0-11e8-9b3b-2bfdf15f92a3.png">


### Notes

I kept it real simple to start. There's a bunch of potential variations on static text we might want to use in some cases, but this keeps consistency with what we have now, so as not to implement solutions we might not need in the immediate future.

## Testing Instructions

- `./scripts/update` or run migrations with `./scripts/manage migrate`
- On the Admin page for Concern, create a new one, ensuring that `indicator` remains set to none:  http://localhost:8100/admin/planit_data/concern/add/
- It should create successfully, and should be available in the API at localhost:8100/api/concern/


Closes #737
